apps/mappings/tasks.py

```diff
@@ -340,23 +340,29 @@ def create_fyle_categories_payload(categories: List[DestinationAttribute], works
     return payload
 
 
-def upload_categories_to_fyle(workspace_id):
+def upload_categories_to_fyle(platform: PlatformConnector, workspace_id):
     """
     Upload categories to Fyle
     """
-    fyle_credentials: FyleCredential = FyleCredential.objects.get(workspace_id=workspace_id)
     qbo_credentials: QBOCredential = QBOCredential.get_active_qbo_credentials(workspace_id)
-    platform = PlatformConnector(fyle_credentials)
-
     qbo_connection = QBOConnector(
         credentials_object=qbo_credentials,
         workspace_id=workspace_id
     )
+    general_settings = WorkspaceGeneralSettings.objects.filter(workspace_id=workspace_id).first()
+
     platform.categories.sync()
     qbo_connection.sync_accounts()
-    general_settings = WorkspaceGeneralSettings.objects.filter(workspace_id=workspace_id).first()
-    qbo_attributes: List[DestinationAttribute] = DestinationAttribute.objects.filter(
+
+    accounts: List[DestinationAttribute] = DestinationAttribute.objects.filter(
         workspace_id=workspace_id, attribute_type='ACCOUNT', detail__account_type__in=general_settings.charts_of_accounts).all()
+    qbo_attributes = accounts
+    if general_settings.import_items:
+        qbo_connection.sync_items()
+        items: List[DestinationAttribute] = DestinationAttribute.objects.filter(workspace_id=workspace_id, display_name='Item', attribute_type='ACCOUNT')
+        if items:
+            qbo_attributes = qbo_attributes | items
```

Comment:
we dont use these operators, you can rather do this
```suggestion
            qbo_attributes.extend(items)
```

Reply:
.extend() doesn't work on queryset I think it works on arrays only. I tried doing this on django shell and it gives out an error. 
<img width="688" alt="Screenshot 2023-04-24 at 6 27 02 PM" src="https://user-images.githubusercontent.com/88420539/234002897-30e773ba-67ee-4336-be52-7d8cdeaca027.png">